test/general-spec.js

```diff
@@ -1,34 +1,17 @@
 var expect = require('expect.js');
-var simple = require('simple-mock');
-var fs = require('fs');
 var testHelper = require('../etc/test-helper.js');
 var errorFormatter = testHelper.validationErrorFormatter;
 
-// Load the contents of the sync function file into a global variable called syncFunction
-/*jslint evil: true */
-eval('var syncFunction = ' + fs.readFileSync('build/sync-functions/test-general-sync-function.js').toString());
-/*jslint evil: false */
-
-// Placeholders for stubbing built-in Sync Gateway support functions.
-// More info: http://developer.couchbase.com/mobile/develop/guides/sync-gateway/sync-function-api-guide/index.html
-var requireAccess;
-var requireRole;
-var requireUser;
-var channel;
-
 describe('Functionality that is common to all documents:', function() {
   beforeEach(function() {
-    requireAccess = simple.stub();
-    requireRole = simple.stub();
-    requireUser = simple.stub();
-    channel = simple.stub();
```

Comment:
As mentioned elsewhere, in PR #62 the function stubs were exposed to the outside world via the test-helper module, so there's no longer any need to stub them separately here.